components/script/dom/window.rs

```diff
@@ -319,10 +319,15 @@ impl Window {
         &self.image_cache_thread
     }
 
+    /// Retrieve unsafely the browsing context
```

Comment:
Add a comment, something like "This can panic if it is called after the browsing context has been discarded".